src/photos/ducks/clustering/albums.spec.js

```diff
@@ -0,0 +1,74 @@
+import CozyClient from 'cozy-client'
+import { saveClustering } from './albums'
+import doctypes from 'photos/targets/browser/doctypes'
+
+const client = new CozyClient({ schema: doctypes })
+client.save = jest.fn().mockImplementation((doctype, album) => {
+  return { data: { _id: '1', _type: doctype, ...album } }
+})
+client.create = jest.fn().mockImplementation((doctype, album) => {
+  return { data: { _id: '1', _type: doctype, ...album } }
+})
+client.mutate = jest.fn()
+
+describe('album', () => {
+  const photos = [
+    {
+      datetime: '2020-01-01',
+      clusterId: 'fakeAlbumId1',
+      id: '1'
+    },
+    {
+      datetime: '2020-01-01',
+      clusterId: 'fakeAlbumId1',
+      id: '2'
+    }
+  ]
```

Comment:
Nit: This "photos" variable can be scoped to the first test.

Reply:
Indeed, fixed with 05c982e